src/QMCHamiltonians/NonLocalECPotential_CUDA.cpp

```diff
@@ -278,8 +280,10 @@ void NonLocalECPotential_CUDA::addEnergy(MCWalkerConfiguration& W, std::vector<R
       }
 #endif
       RatioList.resize(QuadPosList.size());
-      RealType vrad[pp.nchannel];
-      RealType lpol[pp.lmax + 1];
+      if (vrad.size()<pp.nchannel)
```

Comment:
This `if` is not necessary, resize performance the same amount of allocation/deallocation and resizing to smaller size cost nothing.

Reply:
Updated.